src/goto-instrument/README.md

```diff
@@ -26,3 +29,159 @@ developers are advised to copy the directory, remove all files apart
 from `main.*`, `parseoptions.*` and the `Makefile` and use these as the
 skeleton of their application. The `doit()` method in `parseoptions.cpp`
 is the preferred location for the top level control for the program.
+
+Usage Examples
+--------------
+
+### Main usage ###
+
+For most of the transformations, `goto-instrument` takes 3 arguments: 
```

Comment:
It doesn't take three arguments, but one or two arguments + any number of options.